app/models/Organization.ts

```diff
@@ -74,3 +83,74 @@ export const getResourceLocations = (org: Organization): LocationDetails[] => {
     recurringSchedule: org.recurringSchedule,
   }));
 };
+
+// TODO Can we remove this?
+// const verifyItem = (item: Organization|Service, itemType: string) => {
+//   const { id } = item;
+//   const changeRequest = { verified_at: new Date().toISOString() };
+//   return post(`/api/${itemType}s/${id}/change_requests`, { change_request: changeRequest })
+//     .then(response => {
+//       // TODO: Do not use alert() for user notifications.
+//       if (response.ok) {
+//         alert(`${itemType.charAt(0).toUpperCase() + itemType.slice(1)} verified. Thanks!`);
+//       } else {
+//         alert(`Issue verifying ${itemType}. Please try again.`);
+//       }
+//     });
+// };
+
+export const getResourceActions = (
+  resource: Organization,
+  service?: Service,
+  filterActions?: string[],
+) => {
+  const phoneNumber = _.get(resource, 'phones[0].number');
+  const latitude = _.get(resource, 'addresses[0].latitude');
+  const longitude = _.get(resource, 'addresses[0].longitude');
```

Comment:
I think we could replace the `.get` method with the ES6 optional chaining operator (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) to avoid having to import lodash in this file

Reply:
+1, I'm all for getting rid of lodash everywhere, I think it does more harm than good.